dbt_platform_helper/utils/aws.py

```diff
@@ -433,6 +433,11 @@ def start_build_extraction(codebuild_client, build_options):
     return response["build"]["arn"]
 
 
+def start_pipeline_extraction(codepipeline_client, build_options):
```

Comment:
Should this be "start_pipeline_execution"?

Reply:
I think change it. The important thing that this is doing is starting the pipeline execution. Perhaps "start_pipeline_and_return_execution_id" or something similar would also work. Or just "start_pipeline" and describe what it returns in the docstring?

Reply:
Updated to `start_pipeline_and_return_execution_id`